johnsonandjohnson/Bodiless-JS
# Feature/token demo 2023

## Changes
Updated: https://github.com/johnsonandjohnson/Bodiless-JS/pull/1648

## Test Instructions
Storybook (run from vital-demo)

- npm run storybook, test tokens on typogorapy, font-size and header pages.
- some token combinations don't make sense, this is expected
- On header page - content is editable and persisted to session storage, but image uploads are not possible

## Related Issues
